pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -304,12 +309,19 @@ func NewTargetProxy(tcpBindAddress string, tcpBindPort int, libvirtdSocketPath s
 		stopChan:       make(chan struct{}),
 		fdChan:         make(chan net.Conn, 1),
 		listenErrChan:  make(chan error, 1),
+		tlsConfig:      tlsConfig,
 	}
 
 }
 
 func (m *migrationProxy) createTcpListener() error {
-	listener, err := net.Listen("tcp", fmt.Sprintf("%s:%d", m.tcpBindAddress, m.tcpBindPort))
+	var listener net.Listener
+	var err error
+	if m.tlsConfig != nil {
+		listener, err = tls.Listen("tcp", fmt.Sprintf("%s:%d", m.tcpBindAddress, m.tcpBindPort), m.tlsConfig)
+	} else {
+		listener, err = net.Listen("tcp", fmt.Sprintf("%s:%d", m.tcpBindAddress, m.tcpBindPort))
```

Comment:
i think I mentioned this in the previous pr.

what would you think about only allowing a non TLS listener if bindAddr == 127.0.0.1?

Reply:
Done.

Reply:
local migrations don't happen. This 127.0.0.1 link is to get around src libvirt not being able to connect to destination libvirt via a unix socket. We have to create fake tcp endpoint and pipe it through the unix socket locally to get out of the pod... not super great :/

Reply:
I see, thanks for the clarification, still need to pass over migration-proxy code :worried: